AntiCheat/Lang/en_US.resx

```diff
@@ -238,118 +238,118 @@
   </data>
   <data name="msg_FreeBuy_SetMoney" xml:space="preserve">
-    <value>Detected player {player} modifying ship credits({Money})!</value>
+    <value>{player} was deteccted modifying the ship's credits. ({Money})!</value>
   </data>
   <data name="msg_FreeBuy_unlockable" xml:space="preserve">
-    <value>Detected player {player} purchasing unlockable for free!</value>
+    <value>{player} was detected buying a ship unlockable for free.</value>
   </data>
   <data name="msg_game_start" xml:space="preserve">
     <value>&lt;color=green&gt;Anti-Cheat (v{ver}) is enabled in this lobby&lt;/color&gt;</value>
   </data>
   <data name="msg_Gift" xml:space="preserve">
-    <value>Detected player {player} anomaly interact the gift box!</value>
+    <value>{player} was detected causing a gift box to behave abnormally.</value>
   </data>
   <data name="msg_GrabObject" xml:space="preserve">
-    <value>Detected player {player} grabbing objects from too far away! (Player coordinates: {player_position}|Item coordinates: {object_position})</value>
+    <value>{player} was detected grabbing objects from too far away. (Player coordinates: {player_position}|Item coordinates: {object_position})</value>
   </data>
   <data name="msg_InfiniteAmmo" xml:space="preserve">
-    <value>Detected player {player} using infinite ammo!</value>
+    <value>{player} was detected using infinite ammo.</value>
   </data>
   <data name="msg_Invisibility" xml:space="preserve">
-    <value>Detected player {player} are turned invisible! (Player coordinates: {player_position})</value>
+    <value>{player} was detected to be invisible. (Player coordinates: {player_position})</value>
   </data>
   <data name="msg_ItemCooldown" xml:space="preserve">
-    <value>Detected player {player} using {item} without respect the cooldown!</value>
+    <value>{player} was detected using an item faster than its cooldown would allow.</value>
   </data>
   <data name="msg_Jetpack" xml:space="preserve">
-    <value>Detected player {player} anomaly exploded the jetpack!</value>
+    <value>{player} was detected forcing a jetpack explosion.</value>
   </data>
   <data name="msg_Kick" xml:space="preserve">
-    <value>{player} was kicked!</value>
+    <value>{player} was banished!</value>
   </data>
   <data name="msg_KillEnemy" xml:space="preserve">
-    <value>Detected player {player} instantly killed a monster! ({enemyName}, monster HP before: {HP})</value>
+    <value>{player} was detected killing a monster instantly. ({enemyName}, monster HP before: {HP})</value>
   </data>
   <data name="msg_Landmine" xml:space="preserve">
-    <value>Detected player {player} anomaly triggered the landmine!</value>
+    <value>{player} was detected forcing a landmine explosion.</value>
   </data>
   <data name="msg_Map" xml:space="preserve">
-    <value>Detected player {player} using Mini-Map mod!</value>
+    <value>{player} was detected to have a mini-map mod. This can make finding loot much easier.</value>

```

Comment:
yea this part is modified by me, but still u need to remove additional message to only keep original meaning